modules/s2i/bash/artifacts/usr/local/s2i/assemble

```diff
@@ -14,3 +14,52 @@ source "${JBOSS_CONTAINER_JAVA_S2I_MODULE}/maven-s2i-overrides"
 # invoke the build
 maven_s2i_build
 
+#TODO: Clean this up to call the scripts instead
```

Comment:
Yeah :-)

Reply:
Even those this code is destined to be removed in favour of calling the other scripts, I guess this is the bit that's been adapted to function in the container, so I'll leave review comments here (too)